tests/cli_test.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime, timedelta
+
+import pytest
+from flask import current_app
+
+from newdle.core.db import db
+from newdle.models import Newdle
+
+
```

Comment:
A test case with multiple newdles at the beginning where only some of them get deleted would be nice.